extension/src/prototypes/view/ui-containers/SearchAutocompletePanel.tsx

```diff
@@ -201,7 +201,7 @@ export const SearchAutocompletePanel: FC<SearchAutocompletePanelProps> = ({
           filters={filters}
           maxHeight={maxMainHeight}
           onFocus={handleFocus}
-          onBlur={handleBlur}
+          onDeepBlur={handleBlur}
```

Comment:
What does `DeepBlur` mean?

Reply:
I need an event passing to `SearchAutocomplete` (child) which could handle blur on parent `SearchAutocompletePanel`. I was using `onBlur` but that's not correct, child's blur doesn't need to trigger parent's blur in this case, so i give it another name.